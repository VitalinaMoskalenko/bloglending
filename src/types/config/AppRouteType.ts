export enum AppRouteType {
  home = "/",
  postDetail = "/postDetail",
  postDetailsWithId = "/postDetail/:id",
}
