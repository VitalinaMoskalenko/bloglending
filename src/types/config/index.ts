export * from "./FontSizeType";
export * from "./ColorType";
export * from "./AppLocalsType";
export * from "./BreakpointType";
export * from "./AppRouteType";
