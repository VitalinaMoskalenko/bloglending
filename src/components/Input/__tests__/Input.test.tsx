import React from "react";
import renderer from "react-test-renderer";
import { ThemeProvider } from "styled-components";
import Input from "../";
import { theme } from "../../../config";

it("renders Input correctly", () => {
  const commentInput = renderer
    .create(
      <ThemeProvider theme={theme}>
        <Input label="label" value="value" onChange={() => {}} />
      </ThemeProvider>
    )
    .toJSON();

  expect(commentInput).toMatchSnapshot();
});
