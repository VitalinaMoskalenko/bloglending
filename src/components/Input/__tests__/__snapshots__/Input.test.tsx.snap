// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Input correctly 1`] = `
<div
  className="sc-dlnjwi koSwQA"
>
  <p
    className="sc-gtsrHT sc-eCApnc ckjQab jDEXhY"
  >
    label
  </p>
  <input
    className="sc-hKFxyN iMFJdb"
    onChange={[Function]}
    value="value"
  />
</div>
`;
